store/sqlstore/group_supplier.go

```diff
@@ -825,6 +825,8 @@ func (s *SqlSupplier) GetGroupsByChannel(ctx context.Context, channelId string,
 		ON
 			gc.GroupId = ug.Id
 		WHERE
+            gc.DeleteAt = 0
```

Comment:
While creating the commands I discovered that we weren't filtering the groups that have been deleted

Reply:
Not a problem, but I actually already have a fix in place for this [here](https://github.com/mattermost/mattermost-server/pull/10701/files#diff-0b7dcefeb566578492d6be5ea6b2453cR930) (and a similar change pending for channels in another PR).